packages/golden_toolkit/lib/src/multi_screen_golden.dart

```diff
@@ -71,3 +78,24 @@ Future<void> multiScreenGolden(
     );
   }
 }
+
+class _FakeWindowPadding implements WindowPadding {
```

Comment:
Should we just make the FakeWindowPadding public?  anyone that uses this is going to have to do it themselves.

Reply:
I made the contract use EdgeInsets instead